src/ctapipe/reco/impact_utilities.py

```diff
@@ -192,6 +191,5 @@ def create_dummy_templates(
 
                 template_dict[key] = template.T * pe
 
-    filehandler = gzip.open(output_file, "wb")
-    pickle.dump(template_dict, filehandler)
-    filehandler.close()
+    with open(output_file, "wb") as filehandler:
```

Comment:
needs to be `gzip.open` not `open`, the reader expects a gzipped file

Reply:
Tests are failing with "not a gzipped file"

Reply:
Yep, saw it